controller/api/models.py

```diff
@@ -290,8 +290,9 @@ def _start_containers(self, to_add):
         [t.start() for t in create_threads]
         [t.join() for t in create_threads]
         if any(c.state != 'created' for c in to_add):
-            err = 'aborting, failed to create some containers'
+            err = 'aborting, rolling back after failure to create containers'
```

Comment:
personally I'm for keeping the error message the same unless we're actually exposing **why** the scheduler failed to create the containers.